getusermedia.html

```diff
@@ -4856,6 +4856,13 @@ <h2>Methods</h2>
                 <var>successfulSettings</var> as the current settings.</li>
                 <li>Return <code>undefined</code>.</li>
               </ol>
+              <div class="note">
+                <p>If the UA relinquished the device, in particular if the track
+                is [=MediaStreamTrack/muted|muted=], applying the settings does not
```

Comment:
Firefox relinquishes camera on `track.enabled = false`, so I'd prefer:

```suggestion
                <p>If the UA relinquished the device, for instance if the track
                is [=MediaStreamTrack/muted|muted=], applying the settings does not
```

Also, it might be nice to do a \<dfn>relinquish the device\<\dfn> [here](https://w3c.github.io/mediacapture-main/#life-cycle) and do a [=relinquish the device|relinquished the device=]. That would make it clearer what we're referencing without inference through example.

Reply:
Done